examples/mysql-execute-operation/cleanup.bal

```diff
@@ -0,0 +1,13 @@
+import ballerina/sql;
+import ballerinax/mysql;
+
+// Cleans up the database after running the example.
+function cleanup() returns sql:Error? {
+    mysql:Client mysqlClient = check new (user = "root", password = "Test@123");
+
+    // Cleans the database.
+    _ = check mysqlClient->execute(`DROP DATABASE CUSTOMER`);
+
+    // Closes the MySQL client.
+    check mysqlClient.close();
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [2c12211](https://github.com/ballerina-platform/ballerina-distribution/pull/3019/commits/2c122113ca286d8157e77209c8fb0f03811542c0)